lib/foreman_maintain/reporter/cli_reporter.rb

```diff
@@ -1,4 +1,3 @@
-require 'thread'
```

Comment:
Still required for ruby 1.8.7

Reply:
Disabled `Lint/UnneededRequireStatement:` in `.rubocop.yml`

Reply:
What about disabling it just here, with a comment on why?

Reply:
Yeah, I tried that actually but `rubocop` is complaining about it saying `Unnecessary disabling of Lint/UnneededRequireStatement`

```
Offenses:
lib/foreman_maintain/reporter/cli_reporter.rb:1:18: W: Unnecessary disabling of Lint/UnneededRequireStatement.
require 'thread' # rubocop:disable Lint/UnneededRequireStatement
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
157 files inspected, 1 offense detected
RuboCop failed!
```

https://travis-ci.org/theforeman/foreman_maintain/jobs/293000847
https://travis-ci.org/theforeman/foreman_maintain/jobs/293001743

Reply:
Have reported this to https://github.com/bbatsov/rubocop/issues/4940